lib/pacio_test_kit/interactions_test.rb

```diff
@@ -1,6 +1,39 @@
 module PacioTestKit
   module InteractionsTest
     # TODO: All helper methods for interactions tests (CRUD) will be added here.
+    def create_and_validate_resources(resource_list, tag = '')
+      # resource_list is list of objects (don't flatten)
+      resource_list.each_with_index do |resource, index|
+        req_num = index + 1
+
+        # make FHIR resource object
+        case resource_type
+        when resource_type == 'Observation'
+          new_resource = FHIR::Observation.new(resource)
+        when resource_type == 'DeviceUseStatement'
+          new_resource = FHIR::DeviceUseStatement.new(resource)
+        when resource_type == 'DiagnosticReport'
+          new_resource = FHIR::DiagnosticReport.new(resource)
+        else
+          no_resource_created_message(resource_type, req_num)
+        end
+
+        # send object to create on server
+        fhir_create(new_resource)
+
+        # check 1: validate the resource from the request.response itself
+        status = request.response[:status]
+        next unless validate_status(status, req_num)
+        next unless validate_json(request.response[:response_body], req_num)
+        next unless validate_resource_type(request, req_num)
+        # skip test checking id
+        next unless validate_resource_creation_time(request, req_num)
+        next unless validate_resource_body(request.response[:request_body], request.response[:response_body], req_num)
+
+        # check 2: validate the resource was created by performing a read request
+        read_and_validate_resources(request.result_id)
```

Comment:
we won't need to do these. This is the [create interaction requirements](https://build.fhir.org/http.html#create). we will only be testing the `SHALL` requirements.